src/zimdump.cpp

```diff
@@ -393,124 +402,128 @@ void ZimDumper::verifyChecksum()
     std::cout << "no checksum" << std::endl;
 }
 
-int main(int argc, char* argv[])
+static const char USAGE[] =
+R"(zimdump.
+
+    Usage:
+      zimdump <zim_file> info [--ns=N]
+      zimdump <zim_file> list [--details] [--idx=INDEX|(--url=URL [--ns=N])]
+      zimdump <zim_file> dump --dir=DIR [--ns=N] [--redirect]
+      zimdump <zim_file> dump (--idx=INDEX |(--url=URL [--ns=N]))
+
+    Options:
+      --idx=INDEX       The index of the article to list/dump
+      --ns=N            Namespace selection [default: A]
+      -h --help         Show this screen.
+      --version         Show version.
+)";
```

Comment:
Please use the usage text I've give in https://github.com/openzim/zim-tools/issues/70#issuecomment-636773620